search/base.py

```diff
@@ -4,23 +4,23 @@
 from unittest.mock import patch
 
 from django.test import (
-    override_settings,
     TestCase,
 )
 
-from search.indexing_api import recreate_index
+from search import indexing_api
+from search.indexing_api import recreate_index, delete_index
 
 
-class ESTestCase(TestCase):
+class ESTestCaseWithIndex(TestCase):
```

Comment:
naming nitpick: if i see a class that inherits from `ESTestCase`, that suggests to me that it's a test case that uses/tests elasticsearch. to me it makes more sense to rename `ESTestCase` to `MockedESTestCase`, and `ESTestCaseWithIndex` to `ESTestCase`. most/all of the test classes that inherit from the mocked ES class don't care if an ES index exists at all. we only need to mock it to handle calls to our indexing API from signals. in that sense i think it's confusing for a test class that doesn't test any kind of ES interaction to inherit from a class called `ESTestCase`. i understand that rename would touch a ton of files. if you like the idea and pycharm's rename function gets this done with minimal fuss, go for it. i'm happy to review the much-larger PR. otherwise you can ignore